README.md

```diff
@@ -1,34 +1,48 @@
-### Prior to Setup
+# Prior to Setup
 Ensure you have:
-- [Git](https://git-scm.com/download/win) downloadeded and setup on your computer. ```git -v``` in terminal to check.
-- [NodeJS/Npm](https://nodejs.org/en/download/current) downloaded and setup on your computer. ```node -v``` and ```npm -v``` in terminal to check.
+1. [Git](https://git-scm.com/download/win)
+2. [NodeJS/Npm](https://nodejs.org/en/download/current)
+
+Use these commands in the terminal to check:
+```
+git -v
+node -v
+npm -v
+```
 
-### File Changes
-- In [index.html](./src/index.html) adjust the title tag.
-- In [package.json](./package.json) adjust the name attribute.
-- In [package-lock.json](./package-lock.json) adjust the name attribute.
 
 # Setup
 
 ### ENV Varaibles
 ```
-EXAMPLE=****
+REACT_APP_API_BASE_URL=****
 ```
 
-### Ensure you have npm installed
-``` npm -v ```
-``` npm i ```
-``` npm start ```
+### Installing Packages
+Next you want to install the requires javascript packages using npm.\
+Once the install is complete, you can start the application:
+```
+npm i 
+npm start
+```
 
-If NodeJS / npm is not installed, download [here](https://nodejs.org/en/download/)
+If NodeJS / npm is not installed, download [here](https://nodejs.org/en/download/).
 
 
 # Testing
-```npm install cypress -D```
-```npx cypress open```
-
-# Tools
+In order to run the automated tests, ensure the above Setup section has been completed.
 
-### ReactJS
-https://reactjs.org/
-### Cypress
+Simply initiate the tests using npm:
+```
+npm test
+```
```

Comment:
Cypress opens but when I run the ass_property test I get an error: 
![image](https://github.com/rgreenan55/SeniorDesign2023FE/assets/97375107/6832f692-8328-48f3-87c0-f5fab39cf5e1)

I couldnt set this environment variable in the terminal for some reason: REACT_APP_API_BASE_URL=****
So I created a .env file and put in there instead but that didn't solve the issue.